pages/studentPortal/intro/Javascript/sports/instructions.tsx

```diff
@@ -1,77 +1,184 @@
+            <h1 className="font-bold">Challenge One</h1>
+            <Button
+              label={challengeOneVisible ? "-" : "+"}
+              size={"small"}
+              backgroundColor={"blue"}
+              textColor={"yellow"}
+              onClick={() => setChallengeOneVisible((prevValue) => !prevValue)}
+            />
           </div>
-          <div>
-            <p className="text-charmander">Step 2:</p>
+
+          {challengeOneVisible && (
             <p>
-              Within this "pd" array, we can see that the first index of the
-              array in this case index 0, is all the plays that happen in period
-              1.
+              Write a function that returns the player name who was projected to
+              make the most number of free throws.
             </p>
+          )}
+
+          <div className="flex flex-row space-x-4">
+            <h1 className="font-bold">Challenge Two</h1>
+            <Button
+              label={challengeTwoVisible ? "-" : "+"}
+              size={"small"}
+              backgroundColor={"blue"}
+              textColor={"yellow"}
+              onClick={() => setChallengeTwoVisible((prevValue) => !prevValue)}
+            />
           </div>
-          <div>
-            <p className="text-charmander">Step 3:</p>
+
+          {challengeTwoVisible && (
             <p>
-              Note that plays is found in the "pla" array. There are many
-              different types of plays in Basketball. Anything from scoring a
-              bucket, jump-off, misses, are all considered plays. The way we
-              differentiate between different plays is by using the "etype"
-              property. Since we are only concerned about the scored buckets, we
-              will only consider the plays with the etype of 15.
+              Write a function that can take a player from this array, analyze
+              their fantasy stats and return a rating from 1 to 100 for that
+              player. You have creativity here on how you want to come up with
+              that rating but base it off their stats.
             </p>
+          )}
+
+          <div className="flex flex-row space-x-4">
+            <h1 className="font-bold">Challenge Three</h1>
+            <Button
+              label={challengeThreeVisible ? "-" : "+"}
+              size={"small"}
+              backgroundColor={"blue"}
+              textColor={"yellow"}
+              onClick={() =>
+                setChallengeThreeVisible((prevValue) => !prevValue)
+              }
+            />
           </div>
-          <div>
-            <p className="text-charmander">Step 4:</p>
+
+          {challengeThreeVisible && (
             <p>
-              We must now figure out a way to fiter only the plays of etype of
-              15.
+              Write a function that sorts this array based on your rating
+              function and print out the top 10 players. // (hint. use map() to
```

Comment:
can you make the hint hidden by default, and have a show hint button that reveals the hint?